OpenRA.Mods.Common/Widgets/Logic/AssetEditorLogic.cs

```diff
@@ -860,27 +862,26 @@ public void Export()
 			edited = false;
 		}
 
-		void EditActor(string name, string parentName, string fieldName, object value)
-		{
-			edited = true;
-			edited = true;
-			if (actorEdits.TryGetValue(name, out var actor))
-			{
-				if (actor.TryGetValue(parentName, out var fields))
-					fields[fieldName] = value;
-				else
-					actor.Add(parentName, new Dictionary<string, object>() { { fieldName, value } });
-			}
-			else
-				actorEdits.Add(name, new Dictionary<string, Dictionary<string, object>>
-				{
-					{
-						parentName,
-						new Dictionary<string, object>() { { fieldName, value } }
-					}
-				});
-		}
-
+		// void EditActor(string name, string parentName, string fieldName, object value)
+		// {
+		// 	edited = true;
+		// 	edited = true;
+		// 	if (actorEdits.TryGetValue(name, out var actor))
+		// 	{
+		// 		if (actor.TryGetValue(parentName, out var fields))
+		// 			fields[fieldName] = value;
+		// 		else
+		// 			actor.Add(parentName, new Dictionary<string, object>() { { fieldName, value } });
+		// 	}
+		// 	else
+		// 		actorEdits.Add(name, new Dictionary<string, Dictionary<string, object>>
+		// 		{
+		// 			{
+		// 				parentName,
+		// 				new Dictionary<string, object>() { { fieldName, value } }
+		// 			}
+		// 		});
+		// }
```

Comment:
Not strictly a style change though I agree this can't exist anymore as editing actor fields this way would conflict with the YAML Hot Reload, it should be one or the other. I would just remove this code altogether.